content/sensu-go/6.8/observability-pipeline/observe-process/silencing.md

```diff
@@ -379,9 +387,30 @@ expire: 3600
 {{< /code >}}
 {{< /language-toggle >}}
 
+<a id="silence-expireat"></a>
+
+expire_at    | 
+-------------|------ 
+description  | Time at which the entry should be deleted. In seconds since the Unix epoch.<br><br>If the silence is set to expire when a [check resolves][22], the `expire_at` value will be `0`.
```

Comment:
I’m not quite sure this is correct, I believe `expire_at` and `expire_on_resolve` can be set independently